services/w3c/w3c-validation.service.js

```diff
@@ -86,13 +88,29 @@ module.exports = class W3cValidation extends BaseJsonService {
     })
   }
 
-  transform(messages) {
-    return messages.reduce((accumulator, message) => {
-      let { type } = message
-      if (type === 'non-document-error') {
-        throw new NotFound({ prettyMessage: 'target url not found' })
+  transform(url, messages) {
+    if (messages.length === 1) {
+      const { subType, type, message } = messages[0]
+      if (type === 'non-document-error' && subType === 'io') {
+        let notFound = false
```

Comment:
More lines of code but I thought this is the easiest to understand instead of some crazy regex

Reply:
Agreed!